inc/spbc-scanner.php

```diff
@@ -1953,6 +1956,15 @@ function spbc_scanner_get_file_by_id($file_id)
     return $file_info ?: false;
 }
 
+function spbc_scanner_load_more_scan_logs()
```

Comment:
Please, add doc-block and rerun tests.

Reply:
done https://github.com/CleanTalk/security-malware-firewall/pull/444/commits/dc6097193ecabd4c35372180045a6d43a4a97f04